src/Targets/Targets.csproj

```diff
@@ -0,0 +1,20 @@
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+  <Import Project="..\..\build\Targets\ProducesNoOutput.Settings.targets" />
+
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
+    <Nonshipping>true</Nonshipping>
+    <OutDir>$(OutDir)Rules\</OutDir>
```

Comment:
The targets and the rules need to be in the same directory. I could rename to TargetsAndRules